.github/workflows/unittests-gpu.yml

```diff
@@ -81,16 +81,17 @@ jobs:
 
       - name: Upload coverage to Codecov(For pull request)
         if: ${{ (failure() || success()) && (github.event_name == 'pull_request' || github.event_name == 'pull_request_target') }}
-        uses: codecov/codecov-action@v1.0.10
```

Comment:
I think there is no need to remove this, we can just merge the conditions: `${{ (failure() || success()) && (github.event_name == 'pull_request' || github.event_name == 'pull_request_target' || github.event_name == 'push') }}`

Reply:
But it will raise issue when uploading codecov in `pull_request_target` event.